giskard/core/suite.py

```diff
@@ -401,10 +401,6 @@ def run(self, verbose: bool = True, **suite_run_args):
         run_args.update(suite_run_args)
 
         results: List[(str, TestResult, Dict[str, Any])] = list()
-        required_params = self.find_required_params()
-        undefined_params = {k: v for k, v in required_params.items() if k not in run_args}
-        if len(undefined_params):
```

Comment:
there's the same check in `to_unittest` that should be consistent with `run`

Reply:
why not keep the same check but convert an exception to a warning?

Reply:
Yes that's right, a warning make a lot more sense especially with live logs now